config/initializers/config.rb

```diff
@@ -1 +1,6 @@
-Config.setup { }
+Config.setup do |config|
+  config.use_env = true
+  config.env_prefix = 'SETTINGS'
+  config.env_separator = '__'
+  config.env_converter = nil # our keys mix ALLCAPS and downcase
+end
```

Comment:
I can't tell what the separator is - is it a single underscore or a double underscore?  Which one is right?

Reply:
I mention it in the PR description: double-underscore.  It can't be single underscore because our keys contain a single-underscore, like `AUTH_CODE`.

Reply:
So, how does it override a setting with a single underscore?  I guess all the double-underscore are converted to `.`?